File: tests/conformance/fixtures/go/19-edge-zero-retry.go
package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	p := sykli.New()

	// retry(0) should be omitted from JSON (same as not setting it)
	p.Task("test").Run("echo test").Retry(0)

	// retry(2) should appear
	p.Task("flaky").Run("echo flaky").Retry(2)

	p.Emit()
}

File: tests/conformance/fixtures/elixir/19-edge-zero-retry.exs
use Sykli

pipeline do
  task "test" do
    run "echo test"
  end

  task "flaky" do
    run "echo flaky"
    retry 2
  end
end
|> Sykli.Emitter.validate!()
|> Sykli.Emitter.to_json()
|> IO.puts()
